File: java/src/n01_oop_vererbung/Schueler.java
package n01_oop_vererbung;

public class Schueler extends Person {
    public String bildungsabscluss;
    public float notendurchschnitt;

    public Schueler(String name, String gebOrt, String gebDat, String bildungsabschluss, float notendurchschnitt) {
        super(name, gebOrt, gebDat);
        this.bildungsabscluss = bildungsabschluss;
        this.notendurchschnitt = notendurchschnitt;
    }
}

File: dart/src/n01_oop_vererbung/schueler.dart
import 'person.dart';

class Schueler extends Person {
  Schueler(super.name, super.gebOrt, super.gebDat, this.bildungsabschluss, this.notendurchschnitt);

  final String bildungsabschluss;
  final double notendurchschnitt;
}
